package.json

```diff
@@ -111,10 +112,6 @@
       "git add ."
     ]
   },
-  "engines": {
-    "node": "^18.0.0",
-    "yarn": "^3.6.0"
-  },
```

Comment:
Please keep these and remove the other

Reply:
yeah，already add this"engines": {
    "node": "^18.0.0",
    "yarn": "^3.6.0"
  },